src/officehours_api/views.py

```diff
@@ -186,6 +186,12 @@ class MeetingDetail(DecoupledContextMixin, LoggingMixin, generics.RetrieveUpdate
     serializer_class = MeetingSerializer
     permission_classes = (IsAuthenticated, IsHostOrAttendee,)
 
+    def update(self, request, *args, **kwargs):
+        try:
+            super().update(request, *args, **kwargs)
+        except MeetingStartedException as e:
+            return Response({'Meeting Detail': e.message}, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
This seems fine. Maybe a nitpick but I wonder if 409 would be the most appropriate response for this error instead of a generic 400?

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/409

Reply:
Hm, maybe, likely we'd have to change this in other places as well. I'm going to keep it as is for now.